src/coreclr/jit/simd.h

```diff
@@ -182,6 +183,11 @@ struct simd64_t
     }
 };
 
+typedef simd64_t simd_t;
```

Comment:
Unrelated to this change:

simd64_t operator== and != use v256. Should simd32_t use v128:
```
    bool operator==(const simd64_t& other) const
    {
        return (v128[0] == other.v128[0]) && (v128[1] == other.v128[1]);
    }

    bool operator!=(const simd64_t& other) const
    {
        return (v128[0] != other.v128[0]) || (v128[1] != other.v128[1]);
    }
```
?

I don't know if these are perf-critical, or if this would help:
```
    bool operator==(const simd64_t& other) const
    {
        return 0 == memcmp(&f32[0], &other.f32[0], sizeof(*this));
    }
```
and similar for the other operators/types.

Reply:
Fixed to use v128 since.